src/deploy/test_account_block.sql

```diff
@@ -328,4 +344,23 @@ BEGIN
   END IF;
 END $$ LANGUAGE plpgsql;
 
+GRANT EXECUTE ON FUNCTION maevsi_test.account_block_create(UUID, UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.account_block_remove(UUID, UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.account_create(TEXT, TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.account_remove(TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.contact_create(UUID, TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.contact_select_by_account_id(UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.contact_test(TEXT, UUID, UUID[]) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.event_category_create(TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.event_category_mapping_create(UUID, UUID, TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.event_category_mapping_test(TEXT, UUID, UUID[]) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.event_create(UUID, TEXT, TEXT, TEXT, TEXT) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.event_test(TEXT, UUID, UUID[]) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.guest_create(UUID, UUID, UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.guest_test(TEXT, UUID, UUID[]) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.guest_claim_from_account_guest(UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.set_invoker(UUID) TO maevsi_account;
+GRANT EXECUTE ON FUNCTION maevsi_test.unset_invoker() TO maevsi_account;
```

Comment:
```suggestion
GRANT EXECUTE ON FUNCTION maevsi_test.invoker_set(UUID) TO maevsi_account;
GRANT EXECUTE ON FUNCTION maevsi_test.invoker_unset() TO maevsi_account;
```
so that it matches the other lines